### PR TITLE
Fix typo in keras (no issue id)

### PR DESCRIPTION
**Pull request by Whovillage**
> Before starting, please choose the relevant pull request **Labels**, **Reviewers**, and **Assignees**

### Why?

The exercise suggest we use categorical_crossentropy , but the classes in breast cancer data set are actually binary, so binary cross entropy should be used to get the audit answers. 

### Solution Overview

change the wording from latter to former

### CODE REVIEW

**Review by eslopfer — DISMISSED** [review]
Good catch! 

Thanks for your contribution.

LGTM :rocket:
